doc/model_details/HillTononiModels.ipynb

```diff
@@ -221,22 +221,29 @@
     "\n",
     "##### Corrected equations\n",
     "\n",
-    "The equations implemented in NEST are thus\n",
+    "The equations for this channel implemented in NEST are thus\n",
     "\n",
     "\\begin{align}\n",
-    " \\frac{dD}{dt} &= D_{\\text{influx}} - \\frac{D-D_{\\text{eq}}}{\\tau_D} \\\\\n",
-    " D_{\\text{influx}} &= \\frac{D_{\\text{influx,peak}}}{1+ \\exp\\left(-\\frac{V-D_{\\theta}}{\\sigma_D}\\right)} \\\\\n",
-    " m_{DK}^{\\infty} &= \\frac{1}{1 + \\left(\\frac{d_{1/2}}{D}\\right)^{3.5}}\n",
+    "I_{DK} &= - g_{\\text{peak},DK} (m_{DK}^{\\infty}(V,t))^3 (V - E_{DK})\\\\\n",
+    " m_{DK}^{\\infty} &= \\frac{1}{1 + \\left(\\frac{d_{1/2}}{D}\\right)^{3.5}}\\\\\n",
+    " \\frac{dD}{dt} &= D_{\\text{influx}}(V) - \\frac{D-D_{\\text{eq}}}{\\tau_D} = \\frac{D_{\\infty}(V)-D}{\\tau_D} \\\\\n",
+    " D_{\\infty}(V) &= \\tau_D D_{\\text{influx}}(V) + {D_{\\text{eq}}}\\\\\n",
+    " D_{\\text{influx}} &= \\frac{D_{\\text{influx,peak}}}{1+ \\exp\\left(-\\frac{V-D_{\\theta}}{\\sigma_D}\\right)} \n",
```

Comment:
Could you add in the "Note the following" list what you told me about the fact that, despite their ambiguous names, ``D_{influx}`` and ``D_{influx,peak}``
>  represent ion concentration change

If you could elaborate a little on this here, I think it would be a good thing.